pkgs/applications/networking/browsers/webmacs/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, qtbase
+, herbstluftwm
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "webmacs";
+  version = "0.8";
+
+  src = fetchFromGitHub {
+    owner  = "parkouss";
+    repo   = "webmacs";
+    rev    = "d692ae8c2bf82cce854f34cb3d762932fe7fdc8b";
+    fetchSubmodules = true;
+    sha256 = "1hzb9341hybgrqcy1w20hshm6xaiby4wbjpjkigf4zq389407368";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    python3Packages.pyqt5_with_qtwebkit
+    python3Packages.dateparser
+    python3Packages.jinja2
+  ];
+
+  buildInputs = [
+    python3Packages.pygments
+    qtbase
+  ];
+
+  doCheck = false;
+
```

Comment:
Add
```
dontWrapQtApps = true;

  makeWrapperArgs = [
    "\${qtWrapperArgs[@]}"
  ];
```

Reply:
This whole set of changes (AFAICT) is one of the things I have tried already. I have about 3 different attempts which all end up with

````
Traceback (most recent call last):
  File "/nix/store/m9vrb25cp52dq5ccpzx33rgnpn5lj6bk-webmacs-0.8/bin/.webmacs-wrapped", line 11, in <module>
    sys.exit(main())
  File "/nix/store/m9vrb25cp52dq5ccpzx33rgnpn5lj6bk-webmacs-0.8/lib/python3.7/site-packages/webmacs/main.py", line 284, in main
    ], instance_name=opts.instance, profile_name=opts.profile)
  File "/nix/store/m9vrb25cp52dq5ccpzx33rgnpn5lj6bk-webmacs-0.8/lib/python3.7/site-packages/webmacs/application.py", line 151, in __init__
    self.profile.enable(self)
  File "/nix/store/m9vrb25cp52dq5ccpzx33rgnpn5lj6bk-webmacs-0.8/lib/python3.7/site-packages/webmacs/profile.py", line 59, in enable
    h = handler(app)
  File "/nix/store/m9vrb25cp52dq5ccpzx33rgnpn5lj6bk-webmacs-0.8/lib/python3.7/site-packages/webmacs/scheme_handlers/webmacs/__init__.py", line 59, in __init__
    loader=PackageLoader(__name__),
  File "/nix/store/lvx5mn52bw0ly7nc9ad398k1m9hq29ji-python3.7-Jinja2-2.10.1/lib/python3.7/site-packages/jinja2/loaders.py", line 222, in __init__
    from pkg_resources import DefaultProvider, ResourceManager, \
ModuleNotFoundError: No module named 'pkg_resources'
````